python/pyarrow/tests/test_parquet.py

```diff
@@ -32,29 +32,21 @@
 
 import pandas.util.testing as tm
 
+pq = pytest.importorskip('pyarrow.parquet')
```

Comment:
The problem is if the extension is built but `pyarrow.parquet` fails to import due to a dynamic linking error, the whole module gets ignored